git-p4.py

```diff
@@ -144,7 +154,7 @@ def prompt(prompt_text):
     """
     choices = set(m.group(1) for m in re.finditer(r"\[(.)\]", prompt_text))
     while True:
-        response = raw_input(prompt_text).strip().lower()
+        response = input(prompt_text).strip().lower()
```

Comment:
I am having trouble with Py2 raw_input on Windows under MINGW64.  I have submitted a PR that converts this to sys.stdin.readline(), but that change could be applied on top of this.  I don't think this needs a change at this time, but for historical records,  the PR is #698